pkg/logentry/stages/labels_test.go

```diff
@@ -158,7 +158,7 @@ func TestLabelStage_Process(t *testing.T) {
 				t.Fatal(err)
 			}
 			st.Process(test.inputLabels, test.extractedData, nil, nil)
-			assert.Equal(t, test.expectedLabels, test.expectedLabels)
+			assert.Equal(t, test.inputLabels, test.expectedLabels)
```

Comment:
Can you swap these around?

```
func Equal(t TestingT, expected, actual interface{}, msgAndArgs ...interface{})
```

So that `test.expectedLabels` is first and `test.inputLabels` is second, this will make any failure messages a little easier to understand.

Reply:
Done!